src/plugins/navigation_embeddable/public/components/navigation_embeddable_panel_editor.tsx

```diff
@@ -227,11 +214,70 @@ const NavigationEmbeddablePanelEditor = ({
       <EuiFlyoutFooter>
         <EuiFlexGroup responsive={false} justifyContent="spaceBetween">
           <EuiFlexItem grow={false}>
-            <EuiButtonEmpty onClick={onClose} iconType="cross">
+            <EuiButtonEmpty onClick={onClose} iconType="cross" flush="left">
               {NavEmbeddableStrings.editor.getCancelButtonLabel()}
             </EuiButtonEmpty>
           </EuiFlexItem>
-          <EuiFlexItem grow={false}>{saveButtonComponent}</EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup>
+              {!isByReference ? (
+                <EuiFlexItem grow={false} css={{ 'margin-left': 'auto' }}>
+                  <EuiToolTip
+                    repositionOnScroll={false}
+                    position="top"
+                    content={
+                      <p>
+                        {NavEmbeddableStrings.editor.panelEditor.getAddToDashboardButtonTooltip()}
+                      </p>
+                    }
+                  >
+                    <EuiButton
+                      disabled={orderedLinks.length === 0}
+                      isLoading={isSaving}
+                      onClick={async () => {
+                        setIsSaving(true);
+                        await onAddToDashboard(orderedLinks);
+                        setIsSaving(false);
+                      }}
+                    >
+                      {NavEmbeddableStrings.editor.panelEditor.getAddToDashboardButtonLabel()}
+                    </EuiButton>
```

Comment:
We probably want to change this button to say `Update` rather than `Add to dashboard` when editing a by value panel:


https://github.com/elastic/kibana/assets/8698078/044ffbff-e9ba-475f-aeb4-550d0b750cd7

Reply:
I changed this to "Apply" which seems more appropriate.